composer/callbacks/checkpoint_saver.py

```diff
@@ -14,6 +14,8 @@
 from pathlib import Path
 from typing import Any, Callable, Dict, List, Optional, Union
 
+import torch.cuda
```

Comment:
can we just do `import torch` to be simple like elsewhere?

Reply:
No longer needed now that the call has been moved to checkpoint.py